cromwell-drs-localizer/src/main/scala/drs/localizer/downloaders/GetmChecksum.scala

```diff
@@ -0,0 +1,67 @@
+package drs.localizer.downloaders
+
+import cloud.nio.impl.drs.AccessUrl
+import drs.localizer.downloaders.AccessUrlDownloader.Hashes
+
+import java.util.regex.Matcher
+
+sealed trait GetmChecksum {
+  def getmAlgorithm: String
+  def value: String
+  def args: String = {
+    // Backslash any single quotes that will be interpolated into the final result. Also `Matcher#quoteReplacement`
+    // for the extra level of escape on the replacement String per the docs in `String#replaceAll`.
+    val sanitizedValue = value.replaceAll("'", Matcher.quoteReplacement(raw"\'"))
```

Comment:
Do we also need to sanitize for `\`s?

Reply:
I'm starting to think this might be better as an "allow list" of base64  characters instead.